### ISSUE TITLE
toBeObservable errors with of(undefined)

### ISSUE THREAD

**Issue by isthatcentered** [just-jeb/jest-marbles#143]
Hello,

I get a `Cannot read property 'length' of undefined at isCharacter`when running a test where the observable has a value of `undefined``

Here are two examples: 
```typescript
let values = {
    u: undefined,
    n: null,
}
// ✅ Passes with "null"
expect( cold( "n", values ) ).toBeObservable( cold( "n", values ) )
			
// 🛑 Fails with "undefined" -> Cannot read property 'length' of undefined
expect( cold( "u", values ) ).toBeObservable( cold( "u", values ) )
```

I'll see if I can fix that :)

**Comment by just-jeb** [comment]
Should be fixed in `2.4.1`. Please confirm.

**Comment by isthatcentered** [comment]
Perfect, it works 👍
